### PR TITLE
copyright for fossasia added

### PR DESCRIPTION
**Pull request by sch00lb0y**
conf.py_t is configured to show copyrights. please review  @imujjwal96 @pri22296

### CODE REVIEW

**Comment by mariobehling** [comment]
@sch00lb0y Please always ask and assign your co-developers to review PRs to speed up the process. Thanks.

File: templates/conf.py_t [hunk 1]
```diff
@@ -65,7 +65,7 @@ master_doc = '{{ master_str }}'
 
 # General information about the project.
 project = u'{{ project_str }}'
-copyright = u'{{ copyright_str }}'
+copyright = 'FOSSASIA'
```
Comment (pri22296) [inline templates/conf.py_t]: Is this really needed? copyright_str seems to automatically generated by sphinx in the format YEAR, AUTHOR

**Comment by niranjan94** [comment]
@sch00lb0y ... I have to agree with @pri22296 on this. A proper copy right string is already being generate. Why do we have to hardcode this ?

**Comment by pri22296** [comment]
At the most we can expose that to an environment variable for users to configure.

**Comment by niranjan94** [comment]
@pri22296 that is also not needed at this time. We already have an environment variable named `AUTHOR` to configure the name displayed. That is more than enough at this point. 

Closing the PR and the accompanying issue.

**Pull request closed without merging**